server/cron-job.js

```diff
@@ -0,0 +1,54 @@
+const { CronJob } = require('cron');
+const logger = require('./logger.js');
+const { dbClient, collections } = require('./db');
+const { getUnsuccessfulSbcTransactions, updateSbcTransactions } = require('./utils/sbc-phac-queries');
+const { postServiceItem } = require('./utils/service-bc');
+
+const postToSbcAndUpdateDb = async (collection, submission) => {
+  const transaction = await postServiceItem(submission);
+  await updateSbcTransactions(collection, submission.id, transaction);
+  return { id: submission.id, status: transaction.status };
+};
+
+const etsToSbcJob = async () => {
+  const currentDate = new Date().toISOString();
+  logger.info('ets to sbc cron job executed', currentDate);
+  try {
+    const etsCollection = dbClient.db.collection(collections.FORMS);
+    try {
+      let data = await getUnsuccessfulSbcTransactions(etsCollection);
+      // Remove SBC transactions from submission
+      data = data.map((d) => {
+        const { serviceBCTransactions, ...rest } = d;
+        return rest;
+      });
+      const promises = data.map((d) => postToSbcAndUpdateDb(etsCollection, d));
+      // Wait for all SBC and associated DB interactions to complete
+      const results = await Promise.all(promises);
+      logger.info(results, currentDate);
```

Comment:
What gets logged here as the results? Is this just the same as our current script (logging the confirmation #s)?